spec/requests/lobbies_spec.rb

```diff
@@ -230,9 +230,15 @@
   describe 'start' do
     let(:quiz) { FactoryBot.create(:quiz, user: user) }
     let(:lobby) { FactoryBot.create(:lobby, quiz: quiz) }
+    let(:question1) { FactoryBot.create(:question, quiz: quiz) }
```

Comment:
If you don't need to reference these afterwards, you can just call `FactoryBot` in the `before` block 😄